### PR TITLE
CT-802 Add algo and add newkey command

### PR DESCRIPTION
**Pull request by argjv**
In this PR:
- Add a command to generate new key pairs like in `bitgo-cli`.
- Add Algorand

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitgo-cli-v2",
-  "version": "0.9.3",
+  "version": "0.9.4",
```
Comment (tylerlevine) [inline package.json]: New version should be 0.10.0

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "argparse": "1.0.9",
     "bignumber.js": "~6.0.0",
-    "bitgo": "~4.21.0",
+    "bitgo": "7.0.0",
```
Comment (tylerlevine) [inline package.json]: `^7.0.0` please

File: src/bgcl.js [hunk 3]
```diff
@@ -457,6 +464,33 @@ BGCL.prototype.handleFee = co(function *handleFee(opts) {
...
+  const params = opts.correctParams(opts.args);
+
+  if (opts.args.coin) {
+    if (!isValidCoin(opts.args.coin)) {
+      throw new Error(`${coin} is an invalid cointype for selected environment -> ${opts.bitgo.getEnv()}`);
+    }
+    delete params.coin;
+  }
+
+  const seed = crypto.randomBytes(32);
```
Comment (tylerlevine) [inline src/bgcl.js]: Don't think you need to pass in external entropy - if you just don't pass any seed, it will create a random one.

**Review by tylerlevine — CHANGES_REQUESTED** [review]
Looks pretty good

File: src/bgcl.js (hunk 3 above)
Comment (argjv) [reply src/bgcl.js]: Should I pass entropy or the seed? I was thinking about making the seed an optional argument so ppl can recover their pub and priv based on that

**Review by tylerlevine — APPROVED** [review]
lgtm, up to you if you'd like to wait for a review from @zacleids or someone with more experience with the cli

**Pull request merged**